src/Symfony/Component/Validator/Context/LegacyExecutionContext.php

```diff
@@ -42,6 +41,8 @@ class LegacyExecutionContext extends ExecutionContext
      */
     public function __construct(ValidatorInterface $validator, $root, MetadataFactoryInterface $metadataFactory, TranslatorInterface $translator, $translationDomain = null)
     {
+        trigger_error('The '.__CLASS__.'  class is deprecated since version 2.5 and will be removed in 3.0. Use the new Symfony\Component\Validator\Context\ExecutionContext class instead.', E_USER_DEPRECATED);
```

Comment:
This must be removed, because Symfony still instantiate this class all the time to be able to enable the BC layer